public/styles/style-quiz.css

```
@@ -355,4 +355,13 @@ footer p {
         text-align: center;
     }
     
-}
\ No newline at end of file
+}
+
+a:focus {
+    outline: 2px solid var(--bs-secondary-bg);
+}
```

Doesn't work because you chose a transparent background color. I'd also add offset.

```suggestion
a:focus {
    outline: 2px solid var(--bs-secondary);
    outline-offset: 5px;
}
```

This will work and be visible.

Be sure to move this up in de css file though, this does not belong at the very end. I'd place them in the "body" section above, right after the h1 h2 stuff. It's a global element after all.